.github/workflows/ci.yaml

```diff
@@ -46,6 +46,18 @@ jobs:
         run: |
           neuro config login-with-token ${{ secrets.NEURO_TOKEN }} ${{ secrets.NEURO_STAGING_URL }}
           neuro config show
+      - name: Configure GCP access
```

Comment:
Great if this works! however, an easier solution to test `data-cp `local<->cloud` would be to spawn a job from the user `cookiecutter-e2e` that uses already existing secrets `aws` and `gcp` to reach the clouds. The same way we would test data-cp `storage<->cloud`: spawn a job that in turn will spawn a job.

Reply:
but since it works, let's keep it